src/StackAdmin/Azs.AzureBridge.Admin/Module/Azs.AzureBridge.Admin/Azs.Azurebridge.Admin.psd1

```diff
@@ -56,8 +56,8 @@ Licensed under the MIT License. See License.txt in the project root for license
     # ProcessorArchitecture = ''
 
     # Modules that must be imported into the global environment prior to importing this module
-    RequiredModules   = @(@{ModuleName = 'AzureRM.Profile'; ModuleVersion = '5.5.1'; },
-        @{ModuleName = 'AzureRM.Resources'; ModuleVersion = '6.0.2'; })
+    RequiredModules   = @(@{ModuleName = 'AzureRM.Profile'; ModuleVersion = '5.6.0'; },
+        @{ModuleName = 'AzureRM.Resources'; RequiredVersion = '6.0.2'; })
```

Comment:
@bganapa what's the reason to depend on a specific version of `AzureRM.Resources` for these modules?

Reply:
AZURESTACK supports only a particular version of AzureRm.Resources, we have a copy in the stackadmin folder. We cannot take dependency on the latest as the api-version will be that of latest azure.